pkg/api/v1beta2/types.go

```diff
@@ -653,6 +672,8 @@ const (
 	ResourceCPU ResourceName = "cpu"
 	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	ResourceMemory ResourceName = "memory"
+	// CPU Frequency, in Khz
+	ResourceCPUFrequency ResourceName = "cpu_frequency_khz"
```

Comment:
Yes, please move this. It makes no sense as a resource. It would go into NodeSystemInfo if you want to include it.

If we want cpu performance to influence scheduling, it should be via a scaled compute unit, not through constraints.